wapm-toml/src/lib.rs

```diff
@@ -625,18 +658,37 @@ name = "mod"
 source = "target/wasm32-wasi/release/mod.wasm"
 interfaces = {"wasi" = "0.0.0-unstable"}
 
+[[module]]
+name = "mod-with-exports"
+source = "target/wasm32-wasi/release/mod-with-exports.wasm"
+exports = { wit = "exports.wit", wit_bindgen = "0.0.0" }
```

Comment:
`exports` is a term very tied to WIT I believe. If people used other serialization formats (such as protobuf, for example), that naming might not be the best one.

What about naming it `bindings` and `wit-exports` ?

```suggestion
bindings = { wit-exports = "exports.wit", wit_bindgen = "0.0.0" }
```

Reply:
Good point.